app/lib/types.ts

```diff
@@ -119,6 +122,14 @@ export interface PosAndDir extends Position {
   dir: Direction;
 }
 
+export function asPosition({ row, col }: PosAndDir): Position {
```

Comment:
I think maybe this is dead code - don't see it being used anywhere. Don't really see the point to it anyway as anything in TS that takes a Position should take a PosAndDir just fine.

Reply:
Heh nvm I see where it's used now. Is it really necessary, wouldn't just using the PosAndDir directly work too?

Reply:
Removed!